ui-tests/src/test/java/org/openmrs/reference/page/AddOrEditAllergyPage.java

```diff
@@ -23,6 +26,9 @@
     private static By DRUG_ID;
     private static String REACTION_XPATH = "//div[@id='reactions']/ul/li/label[text()=\"";
     private static By REACTION_ID;
+    private static By ALLERGY_NOTE = By.id("allergy-comment");
+    private static By ALLERGIES_LIST = By.cssSelector("#allergies tbody tr td.allergen");
+    private static By RETURN = By.cssSelector("#content > button.cancel");
```

Comment:
this also needs to be more qualified such addition of any extra cancel buttons won't break the test